plugins/bcm57xx/fu-self-test.c

```diff
@@ -17,7 +17,7 @@
 static void
 fu_bcm57xx_create_verbuf(guint8 *bufver, const gchar *version)
 {
-	memcpy(bufver, version, strlen(version) + 1);
+	memcpy(bufver, version, strlen(version) + 1); /* nocheck:blocked */
```

Comment:
Even if it's self test I think it should be fu_memcpy_safe so that other plugins that copy paste code copy a good example.

Reply:
Ohh this is terrible code, will fix :)

Reply:
Fixed.